src/ProductConstructionService/ProductConstructionService.Cli/Operations/DeploymentOperation.cs

```diff
@@ -126,30 +126,36 @@ public async Task<int> RunAsync()
         }
     }
 
+    private async Task RemoveRevisionLabel(string revisionName, string label)
+    {
+        var result = await InvokeAzCLI([
+                "containerapp", "revision", "label", "remove",
+                ],
+            [
+                "--label", label
+            ]);
```

Comment:
The way this is indented seems cursed to me :D

How about this?

```suggestion
        var result = await InvokeAzCLI(
            ["containerapp", "revision", "label", "remove"],
            ["--label", label]);
```

Reply:
yeah this is cursed